packages/synapse-react-client/src/lib/containers/IconSvg.tsx

```diff
@@ -359,6 +360,7 @@ function IconMapping(props: { icon: string; sx?: SxProps }) {
     case 'helpOutlined':
       return <HelpOutlined sx={sx}></HelpOutlined>
     case 'close':
+    case 'cross':
```

Comment:
Same icon but different semantic meaning should make this a bit easier to refactor in the future, if needed.